nilearn/tests/test_masking.py

```diff
@@ -200,26 +200,6 @@ def test_unmask():
             assert_true(t[0].flags["F_CONTIGUOUS"])
             assert_array_equal(t[0], unmasked3D)
 
-    # 5D test
```

Comment:
5D data is not accepted. I think that if we want to accept it, we should do all the proper verifications that comes with it.